Apache-Beam-Jobs.md

```diff
@@ -227,6 +229,257 @@ error_pcoll = (
 
 `Runner`s provide the `run()` method used to visit every node (`PValue`) in the pipeline's DAG by executing the edges (`PTransform`s) to compute their values.  At Oppia, we use `DataflowRunner` to have our `Pipeline`s run on the [Google Cloud Dataflow service](https://cloud.google.com/dataflow).
 
+## Writing Apache Beam Jobs
+
+For this section, we'll walk through the steps of implementing a job by writing one: `CountExplorationStatesJob`.
```

Comment:
Explain what your example job does, so that the contributor knows what they're trying to do and what the problem statement actually is. There should probably be some planning stage at the outset for "how" to do it / how to think through the impl, maybe you can guide them through the general thinking that would be needed as step 0.

Reply:
Done.